### ISSUE TITLE
Encryption and CLI

### ISSUE THREAD

**Issue by xf-** [feschber/lan-mouse#233]
I have existing connections and encryption got added. I looked throw the commit and README and only GUI was referenced for verifying keys and list has no commands like that in CLI.

Can I update existing connections without editing TOML files in CLI?

**Comment by feschber** [comment]
Not right now, but I plan to add it.
I want to rework the cli a little bit so I will probably do it together with that.

**Comment by xf-** [comment]
Yeah, noticed all the changes, all the new feature/module names in cargo

**Comment by xf-** [comment]
> Not right now, but I plan to add it. I want to rework the cli a little bit so I will probably do it together with that.

Hard to debug so far, but the current state is really unstable. Cursors are often looked to a display (client or host). I need to Press Sift+Ctrl+Alt+Meta on a regular basis to release the cursor or be able to move to second screen. Also, I sometimes get 2 cursors, moving parallel. That looks strange. One on client and other one on server.
(I know that is a separate issue, but I have no debug data at the moment)

**Comment by feschber** [comment]
seems like I have to test KDE again

**Comment by xf-** [comment]
Server
cargo build --release --no-default-features --features libei_capture 

Client
cargo build --release --no-default-features --features libei_emulation

I can add more, but I prefer to keep it small

Maybe it is the shake, but I never had that before. e.g., I restarted on client and mouse is captured every time I entered. It feels like keys are pressed/stuck. Had that issue before with high traffic (far below interface ratings and connection speed)

**Comment by feschber** [comment]
> If I add the capture on client as well, it works a lot better.

Could you clarify what you mean by this?

**Comment by feschber** [comment]
> It feels like keys are pressed/stuck. Had that issue before with high traffic (far below interface ratings and connection speed)

I think I have broken the key release when the release bind is pressed.

**Comment by xf-** [comment]
> > If I add the capture on client as well, it works a lot better.
> 
> Could you clarify what you mean by this?

Running this on the client:
cargo build --release --no-default-features --features libei_emulation,libei_capture

Without capture, I was often unable to release the mouse

**Comment by xf-** [comment]
Lol, something is really strange.
If I'm on the client, I will get focus state scrollbars on server, but no cursor or clicks or other issues, but it moves around like a ghost.

**Comment by feschber** [comment]
This is now supported through `lan-mouse cli authorize-key` / `lan-mouse cli remove-authorized-key`
